deployment/helm/synopsys-alert/templates/alertdb.yaml

```diff
@@ -0,0 +1,178 @@
+{{- if not .Values.alertdb.isExternal }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: alert
+    component: alertdb
+  name: {{ .Release.Name }}-alert-postgres
```

Comment:
To all the names, can you please remove _-alert_? This is because already in synopsysctl we are taking care of adding -alert to the release name.

And also the customer who uses Black Duck along with Alert, then need to mandatory add -alert to the release name in order to work. This is because you can't have 2 releases having same names for different apps in the same namespace.